conformance/google-java/src/test/kotlin/com/connectrpc/conformance/Conformance.kt

```diff
@@ -775,7 +806,7 @@ class Conformance(
                     countDownLatch.countDown()
                 }
             }
-            countDownLatch.await(5, TimeUnit.MINUTES)
```

Comment:
Had set this higher while debugging - can revert back to more reasonable wait times.